packages/compass-global-writes/src/components/states/shard-key-correct.tsx

```diff
@@ -0,0 +1,200 @@
+
+const nbsp = '\u00a0';
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[400],
+});
+
+const codeBlockContainerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[100],
+  maxWidth: '700px',
+});
+
+const paragraphStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[100],
+});
+
+export type ShardKeyCorrectProps = {
+  namespace: string;
+  shardKey: ShardKey;
+  shardZones: ShardZoneData[];
+  isUnmanagingNamespace: boolean;
+  onUnmanageNamespace: () => void;
+};
+
+export function ShardKeyCorrect({
+  namespace,
+  shardKey,
+  shardZones,
+  isUnmanagingNamespace,
+  onUnmanageNamespace,
+}: ShardKeyCorrectProps) {
+  const customShardKeyField = useMemo(() => {
+    return shardKey.fields[1].name;
+  }, [shardKey]);
+
+  const { atlasMetadata } = useConnectionInfo();
+
+  const sampleCodes = useMemo(() => {
+    const { collection, database } = toNS(namespace);
+    return {
+      findingDocuments: `use ${database}\ndb.${collection}.find({"location": "US-NY", "${customShardKeyField}": "<id_value>"})`,
+      insertingDocuments: `use ${database}\ndb.${collection}.insertOne({"location": "US-NY", "${customShardKeyField}": "<id_value>",...<other fields>})`,
+    };
+  }, [namespace, customShardKeyField]);
+
+  return (
+    <div className={containerStyles}>
+      <Banner variant={BannerVariant.Info}>
+        <strong>
+          All documents in your collection should contain both the ‘location’
+          field (with a ISO country or subdivision code) and your{' '}
+          {customShardKeyField} field at insert time.
+        </strong>
+        {nbsp}We have included a table for reference below.
+      </Banner>
+
+      <div className={codeBlockContainerStyles}>
+        <Body data-testid="shardkey-description-title">
+          <strong>{namespace}</strong> is configured with the following shard
+          key:
+        </Body>
+        <Code language="js" data-testid="shardkey-description-content">
+          {shardKey.fields.map((field) => `"${field.name}"`).join(', ')}
+        </Code>
+      </div>
+
+      <Subtitle>Example commands</Subtitle>
+      <div className={paragraphStyles}>
+        <Body>
+          Start querying your database with some of the most{' '}
+          <Link
+            href="https://www.mongodb.com/docs/atlas/global-clusters"
+            hideExternalIcon
+          >
+            common commands
+          </Link>{' '}
+          for Global Writes.
+        </Body>
+        <Body>
+          Replace the text to perform operations on different documents. US-NY
+          is an ISO 3166 location code referring to New York, United States. You
+          can look up other ISO 3166 location codes below.
+        </Body>
+      </div>
+
+      <div className={codeBlockContainerStyles}>
+        <Label htmlFor="finding-documents">Finding documents</Label>
```

Comment:
There is no corresponding id attr anywhere in this file, what is this pointing to?